### PR TITLE
Added Bradford MDC - closes #442

### LINKED ISSUE #442: Bradford Council Request
### Name of Council

Bradford

### Example Address/Postcode

BD13 2NB

### Additional Information

https://onlineforms.bradford.gov.uk/ufs/collectiondates.eb?ebd=0&ebp=30&ebz=1_1701123159341

### Verification

- [X] I searched for similar issues at https://github.com/robbrad/UKBinCollectionData/issues?q=is:issue and found no duplicates
- [X] I have provided a tested working address/postcode/UPRN with bin collections available
- [X] I understand that this project is run by volunteer contributors and completion depends on numerous factors - even with a request, we cannot guarantee if/when your council will get a script

### PR DESCRIPTION
**Pull request by jimmygulp**
I've added Bradford Council for bin collections. Used 'Vale of White Horse' council as a base (they seemed to use the same CRM for creating pages).

Currently finds the next collection date and lets you know which bin to put out. Doesn't take into account the subscription option for garden waste, nor does it currently look up the upcoming dates for future collections.

### CODE REVIEW

**Review by dp247 — COMMENTED** [review]
Looks good - just missing an addition to the [feature file](https://github.com/robbrad/UKBinCollectionData/blob/master/CONTRIBUTING.md#feature-file) 😄

**Comment by jimmygulp** [comment]
Feature file has been updated now :)

File: uk_bin_collection/tests/features/validate_council_outputs.feature [hunk 1]
```diff
@@ -71,6 +71,11 @@ Feature: Test each council output matches expected results
 		| council | selenium_url | selenium_mode |
 		| BoltonCouncil | http://selenium:4444  | local  |
 
+        @BradfordMDC
+		Examples: BradfordMDC
+		| council | selenium_url | selenium_mode |
+		| BradfordMDC | http://selenium:4444  | local  |
```
Comment (dp247) [inline uk_bin_collection/tests/features/validate_council_outputs.feature]: You can set the URL and mode to `None` if the council doesn't use Selenium

Comment (jimmygulp) [reply uk_bin_collection/tests/features/validate_council_outputs.feature]: I've updated the file as it's not Selenium, thanks!

**Review by robbrad — APPROVED** [review]

**Pull request merged**